pkg/status/templates/systemprobe.tmpl

```diff
@@ -0,0 +1,27 @@
+{{/*
+NOTE: Changes made to this template should be reflected on the following templates, if applicable:
+* cmd/agent/gui/views/templates/generalStatus.tmpl
+*/}}
```

Comment:
This was copied from other templates so I left it there since they had all left it there. I think it's meant to be a reference if you want to format the page for the GUI.